app/controllers/users_controller.rb

```diff
@@ -89,7 +93,11 @@ def new
   def resend_invitation
     begin
       user = as_team_member(cognito_user: get_user(user_id: params[:user_id]))
-      resend_invite(username: user.email)
+      @temporary_password = create_temporary_password
```

Comment:
this doesn't have to be an instance variable?
```suggestion
      temporary_password = create_temporary_password
```

Reply:
It needs to be an instance var for the `send_invite_email` to use it

Reply:
ah, I feel like it would be better to pass it as an argument to the function, instead of relying on the instance variable be present somewhere... making it an instance variable makes it accessible to the views, which in context of a password is not the best